### DISCUSSION TITLE
Joint Link interaction within GLTF Model

### DISCUSSION THREAD

**Discussion by kavikode** [google/model-viewer#4829] [Q&A]
In Model-viewer, can a specific link or joint within a GLTF model follow the mouse cursor? For example, if I have a 3D Avatar I want that 3D Avatar's hand to follow the mouse cursor. Any ideas please?

Thank you

**Comment by elalish** [comment]
No, but if you fork us you can add this kind of functionality by digging down to our Three.js scene.

**Reply by kavikode** [reply]
thanks, please provide more detail of how this functionality can be added to the scene. which file should I look at?

**Reply by elalish** [reply]
You should probably only attempt this if you're already very familiar with Three.js. If you grab our `$scene` symbol, that is the instance of `ModelScene.ts`, which contains the whole Three.js scene. You may even be able to hack this in externally to MV's code base by updating transforms on those internal nodes you want based on pointer events. Sounds like it'll require a lot of math though. And I can't help outside of our public API, so happy hacking!

**Comment by adamwolf** [comment]
I'd love to see whatever you come up with!
